switchboard/manager.py

```diff
@@ -22,6 +22,7 @@
 # any and all threads. The only exception to read-only is when they are
 # populated on Switchboard startup (i.e., operator.register()).
 registry = {}
+registry_by_namespace = {}
```

Comment:
The original is keyed by `ConditionSet` `get_id` and the new one is by its `get_namespace`, so they are different ways of looking up conditionsets efficiently, based on the need.  It might be possible to change everything to use just one, but I suspect not (some places in code have a namespace and some have an id, and need to do a lookup).